### PR TITLE
Replace travis and appveyor with github ci

### PR DESCRIPTION
**Pull request by simonschoelly**
Maybe `DOCUMENTER_KEY` needs to be set, but I don't have enough access rights here.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          - '1.3' # Replace this with the minimum Julia version that your package supports. E.g. if your package requires Julia 1.5 or higher, change this to '1.5'.
+          - '1' # Leave this line unchanged. '1' will automatically expand to the latest stable 1.x release of Julia.
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+          - x86
```
Comment (matbesancon) [inline .github/workflows/ci.yml]:
```suggestion
        include:
          - version: 'nightly'
            os: ubuntu-latest
            arch: x64
          - version: '1'
            os: ubuntu-latest
            arch: x64
          - version: '1.3'
            os: ubuntu-latest
            arch: x64
          - version: '1'
            os: windows-latest
            arch: x64
```

to reduce a bit CI, only test a subset of the combinations

Comment (simonschoelly) [reply .github/workflows/ci.yml]: Can we expect that the lp solvers work as well on mac-os as on linux? Then we can indeed skip that. Also, should we stop testing 32-bit (x86) from now on?

Comment (matbesancon) [reply .github/workflows/ci.yml]: I left 32 bits just in case since we depend on LP solvers

Comment (matbesancon) [reply .github/workflows/ci.yml]: > Can we expect that the lp solvers work as well on mac-os as on Linux

normally yes, they are tested on all platforms

**Comment by simonschoelly** [comment]
Ok, I will then also add x86 testing for ubuntu

**Pull request merged**